gips/inventory/orm/__init__.py

```diff
@@ -34,7 +34,7 @@ def setup():
     if setup_complete:
         return
     if use_orm():
-        busted_drivers = ('sar', 'sarannual', 'modaod', 'merra', 'daymet', 'cdl')
+        busted_drivers = ('sar', 'sarannual', 'modaod', 'daymet', 'cdl')
```

Comment:
Okay, presuming that's done with `export GIPS_ORM=true`, and presuming `gips_inventory merra` looks right afterwards, I think this is ready to merge :+1: